src/NaturalLanguage/Annotation.php

```diff
@@ -0,0 +1,263 @@
+<?php
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\NaturalLanguage;
+
+use Google\Cloud\CallTrait;
+
+/**
+ * Annotations represent the result of a request against the
+ * [Google Cloud Natural Language API](https://cloud.google.com/natural-language/docs).
+ *
+ * @method sentences() {
+ *     Returns an array of sentences found in the document.
+ *
+ *     Example:
+ *     ```
+ *     foreach ($annotation->sentences() as $sentence) {
+ *         echo $sentence['text']['content'];
+ *     }
+ *     ```
+ *
+ *     @codingStandardsIgnoreStart
+ *     @see https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/annotateText#Sentence Sentence type documentation
+ *     @codingStandardsIgnoreEnd
+ *
+ *     @return array|null
+ * }
+ * @method tokens() {
+ *     Returns an array of tokens found in the document.
+ *
+ *     Example:
+ *     ```
+ *     foreach ($annotation->tokens() as $token) {
+ *         echo $token['text']['content'];
+ *     }
+ *     ```
+ *
+ *     @codingStandardsIgnoreStart
+ *     @see https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/annotateText#Token Token type documentation
+ *     @codingStandardsIgnoreEnd
+ *
+ *     @return array|null
+ * }
+ * @method entities() {
+ *     Returns an array of entities found in the document.
+ *
+ *     Example:
+ *     ```
+ *     foreach ($annotation->entities() as $entity) {
+ *         echo $entity['type'];
+ *     }
+ *     ```
+ *
+ *     @see https://cloud.google.com/natural-language/reference/rest/v1beta1/Entity Entity type documentation
+ *
+ *     @return array|null
+ * }
+ * @method language() {
+ *     Returns the language of the document.
+ *
+ *     Example:
+ *     ```
+ *     echo $annotation->language();
+ *     ```
+ *
+ *     @return string
+ * }
+ */
+class Annotation
+{
+    use CallTrait;
+
+    /**
+     * @var array The annotation's metadata.
+     */
+    private $info;
+
+    /**
+     * Create an annotation.
+     *
+     * This class is created internally by
```

Comment:
Maybe this should be in the class description? I know I put it on the constructor in Vision, but that was before we started using the class description very much. WDYT?